packages/stegcracker/PKGBUILD

```diff
@@ -4,14 +4,14 @@
 pkgname=stegcracker
 pkgver=2.0.7
 pkgrel=1
-pkgdesc='Steganography brute-force utility to uncover hidden data inside files'
+pkgdesc='Steganography brute-force utility to uncover hidden data inside files.'
 arch=('any')
 groups=('blackarch' 'blackarch-stego')
-url="https://github.com/Paradoxis/StegCracker"
+url='https://github.com/Paradoxis/StegCracker'
 license=('MIT')
 depends=('python')
 makedepends=('python-setuptools')
-source=("https://files.pythonhosted.org/packages/bd/9a/c5ee56f0b8d18c4664ae161e2f9b65e5fd0e4c5e5ab04c956386e2c8f5e2/$pkgname-$pkgver.tar.gz")
+source=('https://files.pythonhosted.org/packages/bd/9a/c5ee56f0b8d18c4664ae161e2f9b65e5fd0e4c5e5ab04c956386e2c8f5e2/$pkgname-$pkgver.tar.gz')
```

Comment:
Please don't change it, if you use single quotes here then the pkgver and pkgname variables aren't read. Rever the change in it line.